java/pom.xml

```diff
@@ -0,0 +1,82 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.extism.sdk</groupId>
+  <artifactId>extism</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>extism</name>
+  <url>https://github.com/extism/extism</url>
```

Comment:
The project is lacking some metadata (organization, licensing, description, developer information) etc. that I think is necessary for releasing to maven central.

See here for a more [complete library with complete maven metadata](https://github.com/adorsys/keycloak-config-cli/blob/main/pom.xml#L18)